pgcli/main.py

```diff
@@ -653,6 +664,19 @@ def obfuscate_process_password():
 
     setproctitle.setproctitle(process_title)
 
+def output_fits_screen(output, screen_size):
+    for entry in output:
+        new_line_index = entry.find("\n")
+        if new_line_index == -1:
+            if len(entry) > screen_size.columns:
+                return False
+        else:
+            if new_line_index > screen_size.columns:
+                return False
+            if entry.count("\n") >= screen_size.rows:
```

Comment:
I believe `entry` is each row in the result. So counting the number of `\n` in each row to determine if we have exceeded the screen_size.rows won't work. Is that correct? 

That's why I'm suggesting `if len(output) >= screen_size.rows:` as an alternative.